### ISSUE TITLE
ArenaMaps

### ISSUE THREAD

**Issue by CCCCarsten** [slipcor/pvparena#194]
arenamaps:
    aligntoplayer: true
    showspawns: true
    showplayers: true
    showlives: false

The view of the map don't expand when running through a big arena. Buuuuuuuuuuhhh! ;))

**Comment by slipcor** [comment]
Hang on but shouldnt aligntoplayer cover this? Maybe restart fixes this?

**Comment by slipcor** [comment]
I think the maps are broken atm. If you are using aligntoplayer, the map should move with the player. I have a feeling this is related to #195 , in the sense that this bug pastebin shows the error that:

1) breaks the map assignment completely
2) breaks the arena start (and depenencies) completely.

-----------

Also es scheint echt als ob die Bukkit Maps grade voll im arsch sind. Um kurz genaue Infos zu geben, es gibt keine richtige Möglichkeit, eine eindeutige Karte zu laden die schon belegt ist (mit einer laufenden Nummer) - d.h. z.z. kanns jederzeit passieren dass die Karten die PVP Arena angelegt hat von Minecraft überschrieben werden :/

Wie auch immer bin ich mir sicher, dass das mit Ticket #195 zusammen hängt, denn dieser Fehler macht zwei Probleme:

1) Die Karten-an-Spieler-bind-Funktion geht flöten (das erklärt diesen Fehler hier)
2) Die Startfunktion der Arena wird geschrottet, das beeinflusst alle Module die auf den Arenastart warten.

-------

check jenkins plz: v1.3.2.118 - [ArenaMaps] fix a startup error due to broken Map linking that breaks everything

**Comment by CCCCarsten** [comment]
![map](https://cloud.githubusercontent.com/assets/18105655/15450733/33d35318-1fa4-11e6-8eb1-1929752d1cc7.jpg)

Nothing more to see on the map. ;)
Mehr ist nicht ;).

**Comment by slipcor** [comment]
ja. wie gesagt, ich hab nur das symptom behoben, das darunter liegende Problem ist, dass Bukkit/Spigot den Link zur Karten-Erstellung kaputt gemacht haben. DAs lief in 1.7? 1.8? alles perfekt :( Ich muss mal sehen, ich bezwefel aber dass ich das lösen kann

Yeah as said, my fix was just covering the symptom, not the underlying problem that Bukkit/Spigot broke the way of creating maps. It worked in 1.7/1.8. I will look into it but I am not really sure I can solve this.

**Comment by CCCCarsten** [comment]
Die Karten funktionieren hier aber unter Spigot/Bukkit 1.8.8. Ich habe es eben getestet. Witzig ist auch, dass auch während des PvP-Kampfes genau EINMAL die Karte tatsächlich funktionierte, bevor sie dann auf so ein Minumum reduziert worden ist.

maps are normally running under Spigot/Bukkit 1.8.8, but not while fighting PvP ;)

**Comment by slipcor** [comment]
natürlich klappen DIE KARTEN, aber nichit die karte die ich "im plugin" erstelle, das meine ich ja. :)

Well obviously "those" maps work perfectly fine, just not the ones I created myself with the plugin, that is precisely the issue ;)
